tournamentmanager/tournamentmanager.py

```diff
@@ -221,9 +223,11 @@ async def tournamentset_settings(self, ctx: commands.Context):
         embed.description = "Réglages du module de gestion de tournois."
         embed.add_field(name="Rôles", value=roles_description, inline=False)
         embed.add_field(name="Channels", value=channels_description, inline=False)
+        embed.add_field(name="durée du check in", value=check_time, inline=False)
```

Comment:
Les majuscules stp 😭 
```suggestion
        embed.add_field(name="Durée du Check-in", value=check_time, inline=False)
```